src/index.js

```diff
@@ -1,53 +1,72 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from 'react'
+import ReactDOM from 'react-dom'
 import { BrowserRouter } from 'react-router-dom'
 import './core/bootstrap'
 
-import reportWebVitals from './reportWebVitals';
-import generalRoutes from './routes/general';
+import reportWebVitals from './reportWebVitals'
+import generalRoutes from './routes/general'
 import adminRoutes from './routes/admin'
-import hotelOwnerRoutes from './routes/hotelOwner';
-import userRoutes from './routes/user';
-import { TopSection } from './components';
-import { loadUserInfo, getCookieValue } from './utils';
-import { COOKIE_TOKEN } from './constants';
+import hotelOwnerRoutes from './routes/hotelOwner'
+import userRoutes from './routes/user'
+import { TopSection } from './components'
+import { loadUserInfo, getCookieValue } from './utils'
+import { COOKIE_TOKEN } from './constants'
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles'
+
+const theme = createMuiTheme({
```

Comment:
is it used somewhere lol

Reply:
Yes, in theme provider 
![image](https://user-images.githubusercontent.com/74857860/111599275-67d6a680-87d0-11eb-9735-0640f0c00908.png)

Reply:
Yes, but later :p